### PR TITLE
Adding syndicate cyborgs

### PR DESCRIPTION
**Pull request by Alberyk**
Adding an improved syndicate and a tool that allows people to deploy one, by getting one from the ghost pool.

Work in progress, changes will happen and feedback is welcome.

### CODE REVIEW

**Comment by skull132** [comment]
If it's WIP, please don't use the review required flag. That's only used to signal that a PR is complete and can be reviewed for merging.

File: code/game/gamemodes/antagspawner.dm [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	var/searching = 0
+
+
+/obj/item/weapon/antag_spawner/borg_tele/attack_self(mob/user)
+	user << "<span class='notice'>The syndicate robot teleporter is attempting to locate an available cyborg.</span>"
+	searching = 1
+	for(var/mob/dead/observer/O in player_list)
+		if(!O.MayRespawn())
+			continue
+		if(jobban_isbanned(O, "Syndicate") && jobban_isbanned(O, "Mercenary"))
```
Comment (Arrow768) [inline code/game/gamemodes/antagspawner.dm]: I think you should also check if someone is jobbaned from cyborg, its a cyborg after all

**Comment by Arrow768** [comment]
Have you thought about the following ?
* New Cyborg detected Message thats shown to the AI
* AI being able to see through the borgs cams and able to modify stuff (I did that once as AI and it went so far that they had to shut down the captured borg again)
* Borg showing up on the Camera Console

**Comment by Alberyk** [comment]
Added the borg ban, also; all of what you said is already covered.

File: code/game/gamemodes/game_mode.dm [hunk 2]
```diff
@@ -88,7 +88,8 @@ var/global/list/additional_antag_types = list()
 			new/datum/uplink_item(/obj/item/weapon/aiModule/syndicate, 7, "Hacked AI Upload Module", "AI"),
 			new/datum/uplink_item(/obj/item/device/powersink, 5, "Powersink (DANGER!)", "PS",),
 			new/datum/uplink_item(/obj/item/device/radio/beacon/syndicate, 7, "Singularity Beacon (DANGER!)", "SB"),
-			new/datum/uplink_item(/obj/item/weapon/circuitboard/teleporter, 20, "Teleporter Circuit Board", "TP")
+			new/datum/uplink_item(/obj/item/weapon/circuitboard/teleporter, 20, "Teleporter Circuit Board", "TP"),
+			new/datum/uplink_item(/obj/item/weapon/antag_spawner/borg_tele, 12, "Syndicate Cyborg Teleporter", "SCT")
```
Comment (skull132) [inline code/game/gamemodes/game_mode.dm]: Real quick, how many telecrystals are issued to nukies?

**Comment by skull132** [comment]
Here's one of my main concerns: this can be hardcore used to bypass the "ideal" antag ratio. Specially on nuke ops, where the nukies can buy, how many of these? Two at least, or?

**Comment by Alberyk** [comment]
You still need ghosts, besides, they are hardly equal to another new cop, due to being borg and they lack tools to repair them, unlike the small medbay they have. Also, I see your point and due to this, I will raise the cost to 14, so, only two of those at most are possible.

**Pull request merged**